kinoml/core/proteins.py

```diff
@@ -127,11 +135,36 @@ def from_hgnc(cls, identifier, **kwargs):
 
     @classmethod
     def from_name(cls, identifier, **kwargs):
-        raise NotImplementedError
+        import requests
+        import tempfile
+        from MDAnalysis import Universe
+        from pathlib import Path
+
+        url = f"https://files.rcsb.org/download/{identifier}.pdb"
+        response = requests.get(url)
+
+        with tempfile.NamedTemporaryFile(suffix=".pdb") as temp_file:
+            temp_file.write(response.content)
+            u = Universe(temp_file.name)
+            p = Path(temp_file.name)
+
+            return cls(pname=p.name, metadata={"id": identifier}, universe=u, **kwargs)
 
     @property
     def sequence(self):
-        s = "".join([r.symbol for r in self.residues])
+        from MDAnalysis.lib.util import convert_aa_code
+
+        pdb_seq = []
+        three_l_codes = [convert_aa_code(i) for i in list(AminoAcidSequence.ALPHABET)]
+
+        for r in self.universe.residues:
```

Comment:
I think we need to be careful with multiple chains/segments here. I recall a similar issue in `opencadd`: https://github.com/volkamerlab/opencadd/blob/master/opencadd/structure/superposition/engines/mda.py#L189-L228

Reply:
Do you mean return the sequence along with what chain it belongs to?